website/docs/d/notification_destination.html.markdown

```diff
@@ -37,14 +60,17 @@ resource "newrelic_notification_channel" "foo-channel" {
 
 The following arguments are supported:
 
-* `id` - (Required) The id of the notification destination in New Relic.
+Either of the following two attributes are required, and not both:
+* `id` - (Optional) The id of the notification destination in New Relic.
+* `name` - (Optional) The name of the notification destination.
+
+Optional:
 * `account_id` - (Optional) The New Relic account ID to operate on.  This allows you to override the `account_id` attribute set on the provider. Defaults to the environment variable `NEW_RELIC_ACCOUNT_ID`.
 
 ## Attributes Reference
 
 In addition to all arguments above, the following attributes are exported:
 
-* `name` - The name of the notification destination.
```

Comment:
One final thing I just found - since `name` is no longer a computed attribute (like it previously was), I think I understand `name` would be discarded from the list of exported attributes (we expect `name` to be supplied to the configuration of this data source and not be exported). However, I've a few thoughts 

1. If `name` is removed from the list of exported attributes (as per this documentation change), it needs to be removed from `flattenNotificationDestinationDataSource()` in `structures_newrelic_notification_destinations.go` as the `name` attribute is still accessible from the data source (I just tested it with an `output` block where I am able to print the `name` of the destination)

2. Else, we might want to revert this documentation change to have `name` remain as an attribute that may be exported

I am thinking of (2) over (1), since we currently could have users who could be using this data source to obtain the name, and removing it from the list of exported attributes could break their existing configuration. What do you think 🤔

Reply:
I agree, I'll add it back to the list :)